starboard/evergreen/shared/platform_configuration/configuration.gni

```diff
@@ -14,7 +14,7 @@
 
 import("//starboard/build/config/base_configuration.gni")
 
-sb_is_modular = true
+is_modular_toolchain = true
```

Comment:
Lemme know if this looks wrong @andrewsavage1 @y4vor

Reply:
This should be set at the top level instead of in a platform's configuration, that way it is set regardless of toolchain. Also, I thought that `is_modular_toolchain` was specifically saying it isn't evergreen?

Reply:
"I thought that is_modular_toolchain was specifically saying it isn't evergreen"
> im trying to say
a) evergreen builds are modular
&&
b) modular builds are not fully evergreen
> Before this change: sb_is_modular : fixed a)

if we are consolidating the flags the new flag should somehow be set by evergreen.
Otherwise the flags remain separate ?


hmm , it has to be set somewhere by evergreen . As evergreen configs now depend on this flag being set.
https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:starboard/build/config/BUILDCONFIG.gn;l=17?q=p:COBALT%20sb_is_evergreen

maybe above place i can add
if(sb_is_evergreen) {
is_modular_toolchain = true ?
}

Reply:
Maybe we can have is_modular_toolchain and sb_is_evergreen define another set of flags ?